tests/cartesian_tests/integration_tests/multi_feature_tests/test_debug_backend.py

```diff
@@ -350,3 +350,26 @@ def test_stencil(
     test_stencil(field_in, field_out)
 
     np.testing.assert_allclose(field_out.view(np.ndarray)[1, 1, :], [3, 2, 3, 4])
+
+
+def test_table_access_stencil():
+    table_view = np.ones((4,), dtype=np.float64)
+    field_out = gt_storage.zeros(
+        dtype=np.float64, backend="debug", shape=(4, 4, 4), aligned_index=(0, 0, 0)
+    )
+    table_view[:] = [2, 3, 4, 5]
+
+    @gtscript.stencil(backend="debug")
```

Comment:
random thought: once we bring the debug backend back into gt4py mainline, we should check which tests are specific to the debug backend and which tests should actually be annotated with `@gtscript.stencil(backend=ALL_BACKENDS`  (with potential excludes referencing issues with missing implementation in other backends)

Reply:
I'll log that in right away, I can feel myself forgetting

Reply:
https://github.com/GEOS-ESM/SMT-Nebulae/issues/98